docs/apache-airflow/upgrading-from-1-10/index.rst

```diff
@@ -924,6 +929,23 @@ instead. On top of that, a new dag.callback_exceptions counter metric has
 been added to help better monitor callback exceptions.
 
 
+Migrating to TaskFlow API
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Airflow 2.0 introduced the TaskFlow API to simplify the declaration of Python callable tasks.
+Users are encouraged to replace classic operators with their TaskFlow decorator alternatives.
+For details, see :doc:`/tutorial_taskflow_api`.
+
+============================= ============================================
+Classic Operator              TaskFlow Decorator
+============================= ============================================
+``PythonOperator``            ``@task`` (short for ``@task.python``)
+``PythonVirtualenvOperator``  ``@task.virtualenv``
+``BranchPythonOperator``      ``@task.branch``
+``DockerOperator``            ``@task.docker``
+============================= ============================================
```

Comment:
We also have ``@task.kubernetes`` now

Reply:
it was added in https://github.com/apache/airflow/pull/25663

Reply:
Thanks, let's add that in

Reply:
@kaxil Added in [dfd4327](https://github.com/apache/airflow/pull/25845/commits/dfd43274474c85bd76bf301254e1c1e2d7c7db49)